tests/modules/s3_datasets_shares/test_share.py

```diff
@@ -622,6 +653,37 @@ def add_share_item(client, user, group, shareUri, itemUri, itemType):
     return response
 
 
+def update_share_item_filter(client, user, group, input):
+    q = """
+      mutation updateShareItemFilters($input: ModifyFiltersTableShareItemInput!) {
+        updateShareItemFilters(input: $input)
+      }
+        """
+
+    response = client.query(q, username=user.username, groups=[group.name], input=input)
+
+    print('Response from updateShareItemFilters: ', response)
+    return response
+
+
```

Comment:
We are missing the test for `remove_share_item_filter` for `get_share_item_data_filters` i think it is fine with the test of the update filters function